src/api/components/PostInfo.tsx

```diff
@@ -0,0 +1,27 @@
+import { UserInfo } from './UserInfo';
+import { CommentList } from './CommentList';
+// import { Post, User } from '../../react-app-env';
```

Comment:
remove comments
and you don't have to do import if you write types in react-app-env, just use them